administrator/language/en-GB/lib_joomla.ini

```
@@ -661,6 +661,13 @@ JLIB_INSTALLER_SQL_BEGIN="Start of SQL updates."
 JLIB_INSTALLER_SQL_BEGIN_SCHEMA="The current database version (schema) is %s."
 JLIB_INSTALLER_SQL_END="End of SQL updates."
 JLIB_INSTALLER_SQL_END_NOT_COMPLETE="End of SQL updates - INCOMPLETE."
+JLIB_INSTALLER_TUF_FREEZE_ATTACK="Update not possible because the offered update has expired."
+JLIB_INSTALLER_TUF_DEBUG_MESSAGE="TUF Debug Message: %s"
+JLIB_INSTALLER_TUF_INVALID_METADATA="The saved TUF update information is invalid."
+JLIB_INSTALLER_TUF_NOT_AVAILABLE="TUF is not available for extensions yet."
+JLIB_INSTALLER_TUF_DOWNLOAD_SIZE="The size of the update did not match the expected size."
```

```suggestion
JLIB_INSTALLER_TUF_DOWNLOAD_SIZE="The size of the update downloaded did not match the expected size."
```

perhaps this is clearer

---

@brianteeman "update downloaded"? Or "downloaded update"? Or just "download"?

---

all three are valid. I just wanted to make it clear that it has been downloaded

---

Ok. Just wanted to be sure it's not a typo or something.